### PR TITLE
ZPI-119: Helm and Docker fix

### PR DESCRIPTION
**Pull request by samuelus**

### CODE REVIEW

File: backend/auth/auth_test.go [hunk 1]
```diff
@@ -67,16 +68,19 @@ t.Run("TestExtractRoles", func(t *testing.T) {
...
 		claims := jwt.MapClaims{}
 		_, _ = jwt.ParseWithClaims(tokenStr, &claims, nil)
 		roles, _ := ExtractRoles(&claims)
 		expectedRoles := []string{"manage-account", "manage-account-links"}
+		log.Print(roles)
```
Comment (Nightbaka) [inline backend/auth/auth_test.go:78]:
```suggestion
```

File: backend/main.go [hunk 2]
```diff
@@ -40,11 +45,15 @@ func main() {
...
-		handlers.AllowedOrigins([]string{"*"}), // Otwiera na wszystkie domeny
+		handlers.AllowedOrigins([]string{"*"}),
 		handlers.AllowedMethods([]string{"GET", "POST", "PUT", "DELETE", "OPTIONS"}),
 		handlers.AllowedHeaders([]string{"Authorization", "Content-Type"}),
 	)
+
 	health.ServiceStatus.MarkAsUp()
-	log.Printf("marking application readiness as UP")
-	log.Fatal(http.ListenAndServe(":8080", corsHandler(router)))
+	log.Printf("Marking application readiness as UP")
```
Comment (Nightbaka) [inline backend/main.go:54]: Delete these - duplicate

Comment (samuelus) [reply backend/main.go:54]: Chyba nie wiem gdzie - liveness !- readiness jeżeli to o to chodzi

Comment (Nightbaka) [reply backend/main.go:54]: a ok mb

File: charts/kam/values.yaml [hunk 3]
```diff
@@ -4,19 +4,23 @@ fullnameOverride: ""
...
+    ROLEMAP_NAMESPACE: ""
+    ROLEMAP_NAME: ""
+
 backend:
+  healthPort: 8082
   replicaCount: 1
   image:
-    repository: zpikam/backend
-    pullPolicy: Always
+    repository: backend-naprawa
```
Comment (Ninja00Shadow) [inline charts/kam/values.yaml:20]: To be changed

File: charts/kam/values.yaml [hunk 4]
```diff
@@ -4,19 +4,23 @@ fullnameOverride: ""
...
+    pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: ""
-
-  env:
-    KEYCLOAK_URL: ""
-    KEYCLOAK_CLIENTNAME: ""
-    ROLEMAP_NAMESPACE: ""
-    ROLEMAP_NAME: ""
+    tag: "latest"
```
Comment (Ninja00Shadow) [inline charts/kam/values.yaml:23]: Change later

File: charts/kam/values.yaml [hunk 5]
```diff
@@ -58,14 +53,14 @@ backend:
 frontend:
   replicaCount: 1
   image:
-    repository: zpikam/frontend
-    pullPolicy: Always
+    repository: front-naprawa
```
Comment (Ninja00Shadow) [inline charts/kam/values.yaml:56]: Later

File: charts/kam/values.yaml [hunk 6]
```diff
@@ -58,14 +53,14 @@ backend:
...
+    pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: ""
+    tag: "latest"
```
Comment (Ninja00Shadow) [inline charts/kam/values.yaml:59]: Later

File: charts/kam/values.yaml [hunk 7]
```diff
@@ -85,7 +80,7 @@ ingress:
     - host: "kam.local"
       paths:
         - backend:
-            - path: /api
+            - path: /
```
Comment (Ninja00Shadow) [inline charts/kam/values.yaml:83]: Not sure about this.

File: frontend/Dockerfile [hunk 8]
```diff
@@ -1,21 +1,16 @@
...
-
-ENV BACKEND_SERVICE_HOST=kam-backend
-ENV BACKEND_SERVICE_PORT=8080
+COPY env.sh /docker-entrypoint.d/
+RUN chmod +x /docker-entrypoint.d/env.sh
 
 EXPOSE 80
-
-CMD ["/bin/sh", "-c", "envsubst '$BACKEND_SERVICE_HOST $BACKEND_SERVICE_PORT' < /etc/nginx/templates/nginx.conf.template > /etc/nginx/conf.d/default.conf && exec nginx -g 'daemon off;'"]
+CMD ["/bin/sh", "-c", "for f in /docker-entrypoint.d/*; do [ -x \"$f\" ] && \"$f\"; done; envsubst '$CUM_API_URL' < /etc/nginx/templates/nginx.conf.template > /etc/nginx/conf.d/default.conf && exec nginx -g 'daemon off;'"]
```
Comment (Ninja00Shadow) [inline frontend/Dockerfile:16]: You wrote cum

File: frontend/nginx.conf [hunk 9]
```diff
@@ -11,7 +11,7 @@ server {
     }
 
     location /backend/ {
-        proxy_pass http://${BACKEND_SERVICE_HOST}:${BACKEND_SERVICE_PORT}/;
+        proxy_pass http://${CUM_API_URL}/;
```
Comment (Ninja00Shadow) [inline frontend/nginx.conf:14]: Again

**Review by Ninja00Shadow — CHANGES_REQUESTED** [review]

**Review by Nightbaka — APPROVED** [review]
I got forced

**Pull request merged**